Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/ResourceClasses/Constants.java

```diff
@@ -0,0 +1,9 @@
+package org.firstinspires.ftc.team6220_PowerPlay.ResourceClasses;
+
+public class Constants {
+    public double WheelDiam = 3.73;
+    public double WheelCirc = WheelDiam * Math.PI;
+    public double TicksPerRev = 14.147 * 2 * 19;
```

Comment:
For TicksPerRev = 14.147 * 2 * 19, where did 14.147 come from?  The expression doesn't seem correct.

Reply:
Also where did the 2 and 19 come from?